buildscripts/kokoro/android.sh

```diff
@@ -65,6 +65,10 @@ cp "$BASE_DIR/github/grpc-java/buildscripts/set_github_status.py" "$SET_GITHUB_S
 read -r ignored new_dex_count < \
   <("${ANDROID_HOME}/tools/bin/apkanalyzer" dex references app/build/outputs/apk/release/app-release-unsigned.apk)
 
+set +x
+all_new_methods=`"${ANDROID_HOME}/tools/bin/apkanalyzer" dex packages app/build/outputs/apk/release/app-release-unsigned.apk | grep ^M | cut -f4 | sort`
```

Comment:
This will load the proguard mappings to produce the original method names:

`"${ANDROID_HOME}/tools/bin/apkanalyzer" dex packages --proguard-mapping app/build/outputs/mapping/release/mapping.txt app/build/outputs/apk/release/app-release-unsigned.apk`